CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ This changelog documents all notable user-facing changes of VAST.
 
 ## Unreleased
 
+- 🐞 Partitions containing multiple fields with the same field name suffix could
+  fail to return some query results due to a bug in the type resolver.
+  [#1447](https://github.com/tenzir/vast/pull/1447)
```

Comment:
This is very technical, the changelog should be more geared towards the user.
How about:
```
A query for a field or field name suffix that matches multiple fields of different
types would erroneously return no results.
```

Reply:
I think we agreed not to do changelog wordsmithing in the individual PRs in favor of one big wording pass when we rearrange the changlog for the release.

Reply:
To be fair, that's not really wordsmithing but rather changing the content entirely.